### ISSUE TITLE
failed to pause geo-replication  

### ISSUE THREAD

**Issue by Madhu-1** [gluster/glusterd2#735]
steps to reproduce 

1)create volume
2)create geo-replication
cli157 geo-replication create testvol2  root@192.168.122.157::testvol -v -f
Geo-replication session created successfully

3)try to pause the geo-replication

cli157 geo-replication pause testvol2  root@192.168.122.157::testvol -v -f

ERRO[2018-05-07 14:07:39.448552] geo-replicationPausefailed                    error="Request failed with HTTP Status code 500\nopen /var/run/gluster/gsyncd-testvol2-192.168.122.157-testvol.pid: no such file or directory" volume=testvol2
Geo-replication Pause failed

Error: Request failed with HTTP Status code 500
open /var/run/gluster/gsyncd-testvol2-192.168.122.157-testvol.pid: no such file or directory

the error message should be proper

**Comment by aravindavk** [comment]
Good catch, Thanks. This pid file will get created by gsyncd process once Geo-rep is started(I don't see georep start command). Two fixes required 1) catch this error, 2) Allow pause only when monitor status is Started.

**Comment by Madhu-1** [comment]
@aravindavk is this also fixed in #1055

**Comment by aravindavk** [comment]
> @aravindavk is this also fixed in #1055

not yet
